aws/v11.4.1/release.diff

```diff
@@ -0,0 +1,60 @@
+    version: 1.0.8                                                     version: 1.0.8
+  - name: chart-operator                                             - name: chart-operator
+    version: 0.12.4                                                    version: 0.12.4
+  - componentVersion: 1.16.5                                         - componentVersion: 1.16.5
+    name: cluster-autoscaler                                           name: cluster-autoscaler
+    version: 1.16.0                                                    version: 1.16.0
+  - componentVersion: 1.6.5                                          - componentVersion: 1.6.5
+    name: coredns                                                      name: coredns
+    version: 1.1.8                                              |      version: 1.1.10
+  - componentVersion: 0.5.18                                         - componentVersion: 0.5.18
+    name: external-dns                                                 name: external-dns
+    version: 1.2.1                                              |      version: 1.2.2
+  - componentVersion: 3.5.0                                          - componentVersion: 3.5.0
+    name: kiam                                                         name: kiam
+    version: 1.2.2                                              |      version: 1.3.0
+  - componentVersion: 1.9.5                                          - componentVersion: 1.9.5
+    name: kube-state-metrics                                           name: kube-state-metrics
+    version: 1.0.5                                              |      version: 1.1.0
+  - componentVersion: 0.3.3                                          - componentVersion: 0.3.3
+    name: metrics-server                                               name: metrics-server
+    version: 1.0.0                                              |      version: 1.1.0
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.8.0                                              |      version: 1.9.0
+  - componentVersion: 0.18.1                                         - componentVersion: 0.18.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.2.0                                                     version: 1.2.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.6.0                                                     version: 1.6.0
+  - name: aws-operator                                               - name: aws-operator
+    version: 8.7.0                                              |      version: 8.7.1
```

Comment:
Me seeing the diff feature the first time in action. 

<img width="1056" alt="woah_rage_face" src="https://user-images.githubusercontent.com/552769/86919451-8c9a5a80-c128-11ea-952f-bee37f94dffd.png">